docs/rfc/rfc-5.adoc

```diff
@@ -0,0 +1,121 @@
+:toc: macro
+
+= RFC 5: Emptying and closing wallets
+
+:icons: font
+:numbered:
+toc::[]
+
+== Overview
+
+If a wallet still containing BTC is to be closed,
+the remaining BTC must be transferred to a new wallet if possible.
+
+This applies in the following situations:
+
+- a wallet has fallen below a specified minimum BTC amount
+- a wallet has failed a heartbeat due to losing too many operators
```

Comment:
Has it completely failed a heartbeat or has it provided the heartbeat but with a number of active members below the critical number?

To clarify, I think we have two "critical" numbers. One is the minimum number of active members during the DKG so that the result can be accepted, let's call it C_dkg. The second number is the minimum number of wallet members actively signing the heartbeat, let's call it C_h. 

T < C_h < C_dkg < N. 

If the number of members signing the heartbeat falls below C_h, we can consider the wallet at risk and start moving the funds before T is reached. It is especially important for how do we want to implement undelegations.

Reply:
OK, I see it's described in section 1.2.2. Weak heartbeat and 1.2.3. Failed heartbeat or other operation.